byway/src/main/webapp/home.js

```diff
@@ -0,0 +1,8 @@
+window.onload = function(){
+  fetch('/api/gettrips');
```

Comment:
What is the purpose of this `fetch` if you aren't doing anything with the result?

Reply:
In my next PR, I will use the /api/gettrips servlet to list the users past trips on the home.html page, so I just put it there for now and will add to it next PR

Reply:
Prefer to write `// TODO(username): thing you need to do` rather than no-op code.

Reply:
Ok! changed it